src/rules/order.js

```diff
@@ -240,6 +240,50 @@ function makeOutOfOrderReport(context, imported) {
   reportOutOfOrder(context, imported, outOfOrder, 'before')
 }
 
+function createImportsSorter(order)  {
+  const resultModifier = order === 'desc' ? -1 : 1
+  return function (importA, importB) {
+    let result = 0
+
+    if (importA < importB) {
+      result = -1 * resultModifier
+    } else if (importA > importB) {
+      result = resultModifier
+    }
+
+    return result
+  }
+}
+function mutateRanksToAlphabetize(imported, order) {
+  const groupedByRanks = imported.reduce(function(acc, importedItem) {
+    acc[importedItem.rank] = acc[importedItem.rank] || []
```

Comment:
```suggestion
    if (!Array.isArray(acc[importedItem.rank]) { acc[importedItem.rank] = [] }
```

Reply:
> 

I was just looking into this PR because I can use it, though I wonder for what reason you find the suggested change better? For my personally, what's there is more readable.

Reply:
avoiding unconditional assignment is more correct, which is more important than what's readable.